src/main/java/org/apache/datasketches/filters/quotientfilter/QuotientFilter.java

```diff
@@ -21,46 +21,34 @@
 
 import java.util.ArrayList;
 import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Queue;
 import java.util.Set;
 
+import org.apache.datasketches.common.SketchesException;
 import org.apache.datasketches.filters.common.BitArray;
 import org.apache.datasketches.filters.common.HeapBitArray;
 
 public class QuotientFilter extends Filter {
 
-  int bitPerEntry_;
-  int fingerprintLength_;
+  public static final double LOAD_FACTOR = 0.9;
```

Comment:
Based on our discussion this morning, I think having a DEFAULT_LOAD_FACTOR and then a way for the user to set a different load factor.  Based on characterization studies, the default should probably be a little lower.